link.go

```diff
@@ -334,9 +367,10 @@ Loop:
 					// decrement link-credit after entire message transferred
```

Comment:
That comment is then out of date?

Reply:
It's still in-date :) I ended up removing the sync/atomic calls. Even with those in it was still accurate since atomic does alter the passed in value as well.